src/style.css

```diff
@@ -0,0 +1,19 @@
+.block {
+  height: 100px;
+}
+
+.red {
+  background-color: red;
+}
+
+.green {
+  background-color: green;
+}
+
+.blue {
+  background-color: blue;
+}
+
+.nomargin {
```

Comment:
it is ok to style the body, but the class name is a great idea too. Better to choose the name `margin-0`